repositories/defaultImageRepository.js

```diff
@@ -0,0 +1,56 @@
+var Image = require('../models/defaultImage');
+
+exports.create = function(data) {
+	return Image.create(data);
+};
+
+exports.findById = function(id){
+	return Image.findById(id).lean().exec();
+};
+
+exports.query = function(options){
+	
+
+	var conditions = {};
+
+	if ('type' in options) {
+		conditions.type = options.type;
+	}
+
+	var totalCount = null;
+
+	return Image
+		.count(conditions)
+		.then(function(result){
+			totalCount = result;
+
+			var pageNum = 0;
+			var pageSize = 10;
+
+			if ('pageNum' in options) {
+				pageNum = options.pageNum;
+			}
+
+			if ('pageSize' in options) {
+				pageSize = options.pageSize;
+			}
+
+			var skipped = pageNum * pageSize;
+
+			if (pageSize >= totalCount) {
```

Comment:
I saw multiple times of such pagination code. If there is a bug side or some needs to modify, not easy to have them consistent. Suggest to have a util method encapsulating the codes.